devops/openshift/workers-deploy.yml

```diff
@@ -32,6 +32,8 @@ objects:
                   value: "${DB_SERVICE}"
                 - name: REDIS_HOST
                   value: "${REDIS_SERVICE}"
+                - name: WORKERS_PORT
+                  value: "${PORT}"
```

Comment:
readinessProbe and livenessProbe should be added to dc right?

same for queue-consumers-deploy

Reply:
IT will be added as part of next PR